src/main/kotlin/com/cognifide/gradle/aem/environment/docker/runtime/Toolbox.kt

```diff
@@ -10,10 +10,21 @@ class Toolbox(aem: AemExtension) : Base(aem) {
         get() = NAME
 
     override val hostIp: String
-        get() = aem.props.string("environment.docker.toolbox.hostIp") ?: "192.168.99.100"
+        get() = detectHostIp() ?: aem.props.string("environment.docker.toolbox.hostIp") ?: "192.168.99.100"
+
+    @Suppress("TooGenericExceptionCaught")
+    fun detectHostIp() = try {
+        ProcBuilder("docker-machine").withArg("ip").run()
+                .outputString.trim().takeIf { it.isNotBlank() }
+    } catch (e: Exception) {
+        throw DockerException("Cannot detect Docker host IP (error while executing 'docker-machine ip'). Cause: ${e.message}", e)
+    }
 
     override val safeVolumes: Boolean = true
 
+    override val hostInternalIp: String?
+        get() = aem.props.string("environment.docker.toolbox.hostInternalIp") ?: "10.0.2.2"
```

Comment:
I am not sure if that IP need to be automatically grabbed from somewhere...
I have never got it not working...